homeassistant/components/tplink/__init__.py

```diff
@@ -0,0 +1,62 @@
+"""Component to embed TP-Link smart home devices."""
+import logging
+
+from homeassistant import config_entries
+from homeassistant.helpers import config_entry_flow
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'tplink'
+REQUIREMENTS = ['pyHS100==0.3.2']
+
+
+async def async_setup(hass, config):
+    """Set up the TP-Link component."""
+    conf = config.get(DOMAIN)
```

Comment:
Why are you doing this? When you copy paste code, you need to understand what it does.

Sonos and Cast have this line so that people can still configure the platforms how they used to